app-backup/buttersink/buttersink-0.6.8.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="rsync-like utility for btrfs snapshots"
+HOMEPAGE="https://github.com/AmesCornish/buttersink"
+
+LICENSE="GPL-3"
+SLOT=0
+
+# tests require network access
+RESTRICT="test"
+
+if [[ "${PV}" == 9999 ]] ; then
+	inherit git-r3
+	KEYWORDS=""
+	EGIT_REPO_URI="https://github.com/AmesCornish/buttersink"
+	SRC_URI=""
+else
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="https://github.com/AmesCornish/buttersink/archive/${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/boto
+	dev-python/crcmod
+	dev-python/flake8
+	dev-python/psutil
+	sys-fs/btrfs-progs"
+DEPEND="${RDEPEND}"
+
+python_prepare()
+{
```

Comment:
Put `{` on the line with function name. Even if this syntax works with bash, every other ebuild uses the single-line form, so foilow that.